src/Illuminate/Http/Testing/FileFactory.php

```diff
@@ -40,8 +40,9 @@ public function image($name, $width = 10, $height = 10)
      */
     protected function generateImage($width, $height)
     {
-        return tap(tmpfile(), function ($temp) use ($width, $height) {
-            imagepng(imagecreatetruecolor($width, $height), $temp);
-        });
+        $path = tempnam(sys_get_temp_dir(), 'kys');
```

Comment:
omg I didn't realise that 😰
Definitely that is not what it's meant for. Sorry, I'll remove that.